packages/core/admin/admin/src/content-manager/components/BlocksEditor/index.js

```diff
@@ -37,10 +37,20 @@ const Wrapper = styled(Box)`
   border-radius: ${({ theme }) => theme.borderRadius};
 `;
 
+const withImages = (editor) => {
+  const { isVoid } = editor;
+
+  editor.isVoid = (element) => {
+    return element.type === 'image' ? true : isVoid(element);
+  };
+
+  return editor;
+};
```

Comment:
Can you write why we need to do this in the code? A new engineer would probably not understand :)

Reply:
Yes good thinking, will do